bin/ansible

```diff
@@ -134,9 +134,11 @@ if __name__ == '__main__':
             # Show raw stacktraces in debug mode, It also allow pdb to
             # enter post mortem mode.
             raise
-        have_cli_options = cli is not None and cli.options is not None
+        have_cli_options = False
+        if context.CLIARGS:
+            have_cli_options = True
```

Comment:
Just an FYI, this could be consolidated to something like `have_cli_options = bool(context.CLIARGS)`

Reply:
Done in latest commit